commons/build.gradle

```diff
@@ -0,0 +1,8 @@
+dependencies {
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.4'
```

Comment:
same here. plz check tha Kafka does not have such deps

Reply:
Yes, it has. Changed to the version from Kafka.